set.mm

```diff
@@ -39774,6 +39781,34 @@ disjoint classes (see ~ disjdif ).  Part of proof of Corollary 6K of
       OUNUPAUNBCFGBSZFTZCSZGTZJUNAURULUTUMFUQUCGUSUCRUDUAUEABCDEFGUFUG $.
   $}
 
+  ${
+    srcmpltd.1 $e |- ( ph -> ( C e. A -> ps ) ) $.
+    srcmpltd.2 $e |- ( ph -> ( C e. ( B \ A ) -> ps ) ) $.
+    $( If a statement is true for every element of a class and for every
+       element of its complement relative to a second class, then it is true
+       for every element in the second class.  (Contributed by BTernaryTau,
+       27-Sep-2023.) $)
+    srcmpltd $p |- ( ph -> ( C e. B -> ps ) ) $=
```

Comment:
srcmpltd for "statement relative complement"? Usually `dif` meaning set difference would be used but here the relative part is important, so despite the inconsistent labelling I can't think of anything better (might want to wait to see if someone else has a suggestion)

either way initially srcmpltd (this), prsrcmpltd, f1resveqaeq, and f1resrcmplf1d feel like mathbox theorems (although I could see f1resrcmplf1d being useful)

Reply:
Yeah, I really had no clue how to label many of these. Maybe `rcmplantd` for "relative complement antecedent in deduction form" instead of `srcmpltd`?

`f1resrcmplf1d` was intended to be the non-mathbox theorem that justifies keeping the rest out as well. I am planning on using it to prove that a simple graph with a cycle has at least 3 vertices. (I'm also planning to leave my graph theory proofs in my mathbox to start, so `f1resrcmplf1d` could be moved there without a problem.)